velox/exec/SharedArbitrator.cpp

```diff
@@ -205,6 +205,25 @@ uint64_t SharedArbitrator::shrinkCapacity(
   return freedBytes;
 }
 
+uint64_t SharedArbitrator::shrinkCapacity(
+    MemoryPool* requestor,
+    const std::vector<std::shared_ptr<MemoryPool>>& pools,
+    uint64_t targetBytes) {
+  ScopedArbitration scopedArbitration(requestor, this);
```

Comment:
We can handle this in caller as we might not always call this for a memory pool? Thanks!